src/wiki/getting-started/install-of-alternatives.md

```diff
@@ -59,22 +61,34 @@ If you don't want to search and install these mods manually, then try the [Fabul
 
 [See the installation instructions](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#prism-launcher) for Prism Launcher.
 
-### <img src="https://cdn-raw.modrinth.com/data/BYfVnHa7/7f8dc20fc0edd29fd95819a6f40938be0b9cadfa.png" height="20"> Simply Optimized
+### <img src="https://cdn.modrinth.com/data/BYfVnHa7/icon.png" height="20"> Simply Optimized
 
 [Simply Optimized](https://modrinth.com/modpack/sop) is a modpack designed with just optimization in mind. SO has better out-of-the-box performance than Fabulously Optimized, but it doesn't come with the QoL mods or full OptiFine parity you would see in Fabulously Optimized, so you're expected to add any additional mods you want yourself.
 
+### <img src="https://raw.githubusercontent.com/skywardmc/art/main/additive/logo_512h.png" height="20"> Additive
+
+[Additive](https://modrinth.com/modpack/additive) is a modpack similar to Fabulously Optimized which aims to support nearly all OptiFine features. It's based on Adrenaline for better performance, supports Fabric, and can be installed on a wide range of Minecraft versions.
+
 ### <img src="https://raw.githubusercontent.com/skywardmc/art/main/adrenaline/logo_512h.png" height="20"> Adrenaline
 
-[Adrenaline](https://modrinth.com/modpack/adrenaline) is a Fabric/Quilt modpack which aims to improve performance as much as possible while not changing anything about the vanilla game, adding QOL features, or introducing instability. Like Simply Optimized, it does not come with any OptiFine replacement mods, so you'll have to manually install the features that you want.
+[Adrenaline](https://modrinth.com/modpack/adrenaline) is a Fabric modpack which aims to improve performance as much as possible while not changing anything about the vanilla game, not adding QOL features, and not introducing instability. Like Simply Optimized, it does not come with any OptiFine replacement mods, so you'll have to manually install the features that you want.
```

Comment:
```suggestion
[Adrenaline](https://modrinth.com/modpack/adrenaline) is a Fabric modpack which aims to improve performance as much as possible while not changing anything about the vanilla game and not introducing instability. Like Simply Optimized, it does not come with any OptiFine replacement mods, so you'll have to manually install the features that you want.
```
I think "not adding QOL features" is implied already by "not changing anything about the vanilla game". It also sounds sorta negative-ish IMO